src/monitoring/geyser_listener.py

```
@@ -20,26 +20,31 @@
 class GeyserListener(BaseTokenListener):
     """Geyser listener for pump.fun token creation events."""
 
-    def __init__(self, geyser_endpoint: str, geyser_api_token: str, pump_program: Pubkey):
+    def __init__(self, geyser_endpoint: str, geyser_api_token: str, geyser_auth_type: str, pump_program: Pubkey):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)